### DISCUSSION TITLE
初始化项目的example单词是不是拼错了，拼成exmaple

### DISCUSSION THREAD

**Discussion by youwei997** [midwayjs/midway#3892] [General / 闲聊]
<img width="542" alt="屏幕截图 2024-06-06 194244" src="https://github.com/midwayjs/midway/assets/45760135/67f28beb-e62a-4fca-80e1-79ca110d48f5">

**Comment by czy88840616** [comment]
恩，是的

**Reply by czy88840616** [reply]
已处理
create-midway@1.3.1
